src/BuildValues.props

```diff
@@ -8,6 +8,6 @@
       prerelease version number and without the leading zeroes foo-20 is
       smaller than foo-4.
     -->
-    <RevisionNumber>00105</RevisionNumber>
+    <RevisionNumber>00106</RevisionNumber>
```

Comment:
You shouldn't edit this revision directly it happens automatically as part of our official build.